_layouts/dataset.html

```diff
@@ -49,7 +52,7 @@
       <dd>
         {% if page.category.size %}
           {% for category in page.category %}
-            <a href="{{ site.baseurl }}/datasets/?category={{ category | slugify }}">{{ category }}</a><br>
+            <a href="{{ site.baseurl }}/datasets/?category={{ category | slugify }}"><h4><span class="label label-danger">{{ category }}</span></a></h4><br>
```

Comment:
Yea i noticed this when testing a dataset with multiple categories... wondering if a button class might be friendlier as that too with render multiple categories inline?